### PR TITLE
add compute_analysis and notify_done

### PR DESCRIPTION
**Pull request by Meryl-Fang**

### CODE REVIEW

File: assignment_template/Analysis.py
Comment (MariaSemeniuk) [inline assignment_template/Analysis.py:185]: This will have to come from the config. Probably this pull request will have to be modified after the config part is done.  I will add the config with the init work soon.

Comment (MariaSemeniuk) [reply assignment_template/Analysis.py:185]: I'll add ntfy topic name to user config soon.

File: assignment_template/Analysis.py
Comment (MariaSemeniuk) [inline assignment_template/Analysis.py]: Have you seen what the data frame looks like from what I sent you? Want to do more than just 1 mean? Add max, min, mean to height and weight

Comment (MariaSemeniuk) [reply assignment_template/Analysis.py]: and count number of types

**Review by MariaSemeniuk — CHANGES_REQUESTED** [review]
Please see in line comments. 

Unfortunately you are blocked on the notify until I merge the init work into main. 

You can enhance the compute analysis in the mean time.

File: assignment_template/Analysis.py
Comment (Meryl-Fang) [reply assignment_template/Analysis.py]: I could add more stats.
Also,example data would look like this- so I think the code should be okay, but I'll add test
```
{'base_experience': 64,
 'height': 7,
 'id': 1,
 'species': 'bulbasaur',
 'type1': 'grass',
 'type2': 'poison',
 'weight': 69}
```

**Pull request closed without merging**
